fix/fixer_iso_checksum_type_and_url.go

```diff
@@ -0,0 +1,55 @@
+package fix
+
+import (
+	"github.com/mitchellh/mapstructure"
+)
+
+// FixerISOChecksumTypeAndURL is a Fixer that remove the "iso_checksum_url" and
+// "iso_checksum_type" to put everything in the checksum field.
+type FixerISOChecksumTypeAndURL struct{}
+
+func (FixerISOChecksumTypeAndURL) Fix(input map[string]interface{}) (map[string]interface{}, error) {
+	// Our template type we'll use for this fixer only
+	type template struct {
+		Builders []map[string]interface{}
+	}
+
+	// Decode the input into our structure, if we can
+	var tpl template
+	if err := mapstructure.Decode(input, &tpl); err != nil {
+		return nil, err
+	}
+
+	// Go through each builder and replace the iso_md5 if we can
+	for _, builder := range tpl.Builders {
+		checksumUrl := stringValue(builder["iso_checksum_url"])
```

Comment:
To confirm remote files are expected to have the checksum set to something like `file:http://example.com/sha256.sum`, is that correct?

Reply:
Yes, that's a tad confusing but yes, I'll add more examples like Megan suggested

Reply:
weirdly enough a url can be: `file:file://./path/to/file.sum`